### ISSUE TITLE
Обновить скриншоты школ

### ISSUE THREAD

**Issue by elisad5791** [Hexlet/hexlet-comparator#77]
Сделать новые скриншоты для всех школ на MacBook с дисплеем retina. Соотношение сторон скриншота должно быть 16:10.

**Comment by buravlev-arthur** [comment]
Взял в работу

**Comment by buravlev-arthur** [comment]
Взял в работу

**Comment by antonDedyaev** [comment]
Если актуально, могу обновить.

**Comment by acidmaksim** [comment]
@antonDedyaev У тебя mac retina? Если да, то нужно просмотреть все школы и где они растянуты обновить на нормальные.

**Comment by antonDedyaev** [comment]
Да, MacBook Air 2022 с retina.
Ок, беру!

**Comment by antonDedyaev** [comment]
@acidmaksim Скрины делать аналогичные, только с правильным разрешением. Дополнительных не делать, я правильно понимаю?

**Comment by acidmaksim** [comment]
@antonDedyaev да, аналогичные. Где то можно и дополнить, главное по теме

**Comment by rizhik356** [comment]
Актуально? могу сделать

**Comment by ssssank** [comment]
Конечно, мы всегда рады помощи, можете браться за любые открытые задачи :)
